services/storage/tests/unit/test_handlers_simcore_s3.py

```diff
@@ -317,6 +318,7 @@ async def test_copy_folders_from_valid_project_with_one_large_file(
                 expected_file_size=src_file.stat().st_size,
                 expected_upload_id=None,
                 expected_upload_expiration_date=None,
+                expected_sha256_checksum=None,
```

Comment:
SAN mentioned, you might want to check that this is copied as well if exists.

Reply:
Yep, I fixed this test